indexers/src/main/java/org/mousephenotype/cda/indexers/AnatomyIndexer.java

```diff
@@ -143,10 +140,7 @@ public RunStatus run() throws IndexerException, SQLException, IOException, SolrS
                     ma.setAltAnatomyIds(bean.getAltIds());
                 }
 
-                // Set collections.
-                OntologyTermHelper sourceList = new OntologyTermHelperMa(maOntologyService, bean.getId());
-
-                ma.setAnatomyTermSynonym(sourceList.getSynonyms());
+                ma.setAnatomyTermSynonym(maOntologyService.getSynonyms(bean.getId()));
```

Comment:
introduce variable for bean.getId() to be more descriptive and match what's done below for emapaId (see line 308)